hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -869,8 +883,57 @@ private void restart(
     *
     =================================================================================================================*/
 
-    private void reconnect() {
-        // No-op
+    /**
+     * The initialization needed for reconnect. It constructs all schemas appropriately.
+     * These are exactly the same steps done as restart trigger.
```

Comment:
Is it truly the _exact_ same steps? If so, why not just call `restart()` here?

Reply:
They are truly same steps, except the InitTrigger field used in dagger is different. Agreed, I will re-use the same code. Thanks Michael